### ISSUE TITLE
Color contrast failure on password visibility issues in Gravity Forms in dark mode

### ISSUE THREAD

**Issue by amberhinds** [wpa11yday/conference#168]
<img width="1062" height="135" alt="Image" src="https://github.com/user-attachments/assets/a7a528b4-8767-47dc-9242-22a9e7289b84" />

You can see it here: https://wpaccessibility.day/template/organizer-onboarding/

(Must be logged out)
